mix.exs

```diff
@@ -12,7 +12,8 @@ defmodule LambdaEthereumConsensus.MixProject do
       elixirc_paths: compiler_paths(Mix.env()),
       warn_test_pattern: "_remove_warning.exs",
       preferred_cli_env: [
-        dialyzer: :test
+        dialyzer: :test,
+        generate_spec_tests: :test
```

Comment:
this is the main change, running the task in :test env so that the runners are included as part of the compilation.